hack/local-up-cluster.sh

```diff
@@ -908,6 +914,25 @@ function start_kubedns {
     fi
 }
 
+function start_nodelocaldns {
+    if [[ "${ENABLE_NODELOCAL_DNS}" = true ]]; then
```

Comment:
needs two = and a default variable expansion.

Reply:
Also move this to the main.

Reply:
done